.github/workflows/ci.yaml

```diff
@@ -120,3 +121,27 @@ jobs:
           BUILDER_NAME: ${{ matrix.builder }}
         run: |
           make ${IMAGE_NAME} BUILDER=${BUILDER_NAME}
+
+  envtest:
+    name: envtest
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: ${{ env.GO_VERSION }}
+      - uses: actions/cache@v2
+        with:
+          path: |
+                ~/.cache/go-build
+                ~/go/pkg/mod
+          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+          restore-keys: |
+            ${{ runner.os }}-go-
+      - name: Install controller-runtime and get envtest binaries
+        run: |
+          go install sigs.k8s.io/controller-runtime/tools/setup-envtest@latest
```

Comment:
Doesn't this line install envtest binaries? If so, why do we need to download and unpack them?

Reply:
Does it..?
I cannot see where the binaries are..
https://github.com/hj-johannes-lee/intel-device-plugins-for-kubernetes/runs/4256699647?check_suite_focus=true#step:5:3

And also,, there is an issue of using 1.22 kubernetes version.. so I thought it would be better to get the binaries just separately.
https://github.com/kubernetes-sigs/controller-runtime/issues/1571#issuecomment-914395168

Reply:
Why did you close this PR?

as for setup-envtest tool, it can manage multiple versions of binaries, e.g.
```
$ setup-envtest list 
(installed)  v1.22.1  linux/amd64
(available)  v1.22.1  linux/amd64
(available)  v1.22.0  linux/amd64
(available)  v1.21.4  linux/amd64
(available)  v1.21.2  linux/amd64
(available)  v1.20.2  linux/amd64
(available)  v1.19.2  linux/amd64
(available)  v1.19.0  linux/amd64
(available)  v1.17.9  linux/amd64
(available)  v1.16.4  linux/amd64
(available)  v1.15.5  linux/amd64
(available)  v1.14.1  linux/amd64
(available)  v1.13.1  linux/amd64
(available)  v1.12.3  linux/amd64
(available)  v1.11.0  linux/amd64
(available)  v1.10.1  linux/amd64

```

Please, read its documentation for further details: https://pkg.go.dev/sigs.k8s.io/controller-runtime/tools/setup-envtest#section-readme

Reply:
Thank you for your help.! I think i should have just read the setup-envtest readme in more detail.!

I.. closed for a while since I just wanted to forced-push to my branch to see whether it works in github actions or not and that will create so many actions here too (though I think now that I could have just made another temporary branch..!)

I thought I can open again, but I guess,, I need to just make another pr after forced-push to the commit.

Reply:
> I thought I can open again, but I guess,, I need to just make another pr after forced-push to the commit.

Looks like you pushed and then closed. To just re-open BEFORE pushing something again should keep this PR working.

Reply:
I see. I didn't know about that. Let me try not to make an unnecessary pr from now on!